.github/pull_request_template.md

```diff
@@ -1,21 +1,37 @@
 ---
 name: Pull Request
-about: A simple pull request template.
-issue: Issue # if applicable
+about: Request to pull code changes into the repository.
+title: ''
+assignees: ''
 
 ---
 
-Provide a simple description of what the PR achieves. 
+## Relevant Issues
+<!-- List relevant issue tickets here. -->
+<!-- Say "Closes #0" for issues that the PR resolves, replacing 0 with the issue number. -->
 
-## Pull Request Details
 
-Please ensure your PR fulfills the following criteria:
+## Description
+<!-- Describe the changes made in the pull request. -->
 
-- [ ] Have you joined the [PythonDiscord Community](https://pythondiscord.com/invite)?
-- [ ] Were your changes made in a Pipenv environment?
-- [ ] Does flake8 pass (```pipenv run lint```)
 
+## Reasoning
```

Comment:
This could be redundant if the PR is associated with an issue since some issue templates having the same section. It could also just not really make have any value in the case of a bug fix PR.

Reply:
Reasonings for certain changes may change from the initial issue ticket discussion and during the actual implementing stage due to not thought of considerations, testing, etc. Plus not all issues will be fully detailing the reasoning of a certain implemention, instead those sections are just a chance to propose their idea of an implementation. Less specific and not actually detailed.